google/auth/_default.py

```diff
@@ -114,7 +116,7 @@ def load_credentials_from_file(filename, scopes=None):
         try:
             credentials = credentials.Credentials.from_authorized_user_info(
                 info, scopes=scopes
-            )
+            ).with_quota_project(quota_project_id)
```

Comment:
Can not find anywhere that `load_credentials_from_file` is called with a `quota_project_id` parameter.  
So `quota_project_id` seems always be the default `None`.  
At the same time, `quota_project_id` loaded in `info` (from json file) could have work well but overwrote by the `.with_quota_project(quota_project_id)`

Reply:
Ah yes, it looks like I introduced a bug. Will open a PR shortly. Thanks for the review!